### DISCUSSION TITLE
Different Format (Day / Month) for timeAxisMarkers

### DISCUSSION THREAD

**Discussion by mikeboe** [samuelarbibe/dnd-timeline#50] [Q&A] [answered]
Hey there, I'm trying out dnd-timeline as I like it quite a lot. However, I need to change the timeAxisMarkers from hours of the day, to have a monthly view by days, or a yearly view by months. I cannot really get my head around how the logic for them exactly works with the different markers. Do you have a suggestion here for me? Thanks man

**Comment by samuelarbibe** [comment, marked as answer]
Hey Mike!  
First of all, the markers are not part of the library, but an exemplary implementation.  For that reason, there is no official documentation for it.  
That being said, I have already discussed about it in another discussion linked here:  
https://github.com/samuelarbibe/dnd-timeline/discussions/24  
I suggest you to read the thread, and feel free to contact me here if you have further questions.    

Have fun 🤩
